milestones/management/__init__.py

```diff
@@ -1,3 +1,5 @@
 """
 Milestones management package initialization module
 """
+
+from __future__ import unicode_literals
```

Comment:
Any reason to include this but not `absolute_import`?

Reply:
python-modernize won't put in the absolute_import line if there are no imports in the file.  I think it put in the unicode_literals import because of the docstring.